tests/python/tests/test_spark.py

```
@@ -1635,6 +1635,60 @@ def test_encryption_key_id_immutable(spark, namespace):
+
+    # Query with different cases
+    pdf = spark.sql(f"SELECT * FROM {namespace.spark_name}.my_view").toPandas()
+    assert pdf["id"].tolist() == [1]
+
+    pdf = spark.sql(f"SELECT * FROM {namespace.spark_name}.MY_VIEW").toPandas()
+    assert pdf["id"].tolist() == [1]
+
+    spark.sql(f"DROP VIEW {namespace.spark_name}.MY_VIEW")
+
+    # Verify the view is gone
+    with pytest.raises(Exception):
```

Ruff (0.15.9)

[warning] Do not assert blind exception: `Exception`

(B017)